sdk/keyvault/keyvault-certificates/src/index.ts

```diff
@@ -298,7 +298,7 @@ export class CertificatesClient {
       this.pipeline = pipelineOrOptions;
     }
 
-    this.client = new KeyVaultClient(credential, this.pipeline);
+    this.client = new KeyVaultClient(credential, "7.0", this.pipeline);
```

Comment:
I wonder if we should make this a constant in this file with a comment saying that we are using it temporarily until an issue with `autorest.typescript` is fixed?